translate/src/modules/search/components/SearchPanel.tsx

```diff
@@ -31,9 +31,18 @@ const SearchOption = ({
   selected: boolean;
   onToggle: () => void;
 }) => {
+  // Find the first option that doesn't have "Match" in the name, and apply top margin for visual seperation
```

Comment:
At risk of asking a dumb question, this feels brittle. 

Can we have IDs associated to these checkboxes, and add a margin to that element?

Reply:
So for example, `Include string identifiers` would get ID = 3, and then we apply the top margin to the element whose ID = 3? Or did I understand incorrectly

Reply:
Not necessarily a numeric ID, a CSS ID (e.g. `id="option-include-ids"`). Let's wait for @mathjazz though, as I'm not sure if the space is wanted (it made sense when there was text between the two group of options).

Reply:
My deleted (didn't notice this thread earlier) comment:

"That looks complex and fragile. I'd just use slug as the className and add margin to .search_identifiers.

Alternatively you could have two lists, once for Matches and one for Incudes/Excludes."

Reply:
Much clearer now, thanks!

Yeah, it's OK if some classes are unused.